libs/langchain-community/langchain.config.js

```diff
@@ -311,6 +314,7 @@ export const config = {
     "vectorstores/zep",
     "chat_models/bedrock",
     "chat_models/bedrock/web",
+    "chat_models/friendli",
```

Comment:
You don't need to add your models here since they don't use any peer dependencies.

Reply:
I removed "llms/friendli" and "chat_models/friendli" from `requiresOptionalDependency`.